_automating-configurations/api/get-workflow-steps.md

```diff
@@ -28,36 +25,52 @@ OpenSearch validates workflows by using the validation template that lists the r
     ]
   }
 }
-```
-
-The Get Workflow Steps API retrieves this file.   
+``` 
 
 ## Path and HTTP methods
 
 ```json
 GET /_plugins/_flow_framework/workflow/_steps
+GET /_plugins/_flow_framework/workflow/_step?workflow_step=<step_name>
 ``` 
 
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| `workflow_step` | String | The step name of the step to retrieve. Specify multiple step names as a comma-separated list. For example, `create_connector,delete_model,deploy_model`. |
```

Comment:
Can the first instance of "step" be removed? Ignore if essential to the meaning.